federation/index.md

```diff
@@ -44,6 +44,30 @@ but is not yet recommended for processing medium size to large countries or cont
 
 The collection metadata of the Terrascope back-end tries to clearly identify which collections are served by SentinelHub.
 
+#### Commercial Data
+
+openEO Platform provides direct access to commercial data. Currently, data must be purchased directly through Sentinel Hub (see Sentinel Hub documentation on purchasing commercial data [here](https://docs.sentinel-hub.com/api/latest/api/data-import/)), but we are working to support ordering commercial data directly from the platform in the future. 
+
+Data is accessed as part of the load_collection process and via a 'featureflag' argument. To access the data, you must:
+- select the commercial data provider in `collection_id` (e.g. collection_id="PLANETSCOPE")
+- set the Sentinel Hub BYOC collection ID (byoc-{id}) as featureflag argument (e.g. datacube._pg.arguments['featureflags'] = {'byoc_collection_id': byoc_collection_id})
+
+Full example of loading a commercial data collection:
+
+```python
+toc = connection.load_collection(
+    collection_id="PLANETSCOPE",
+    spatial_extent={"west": 104.86, "south": 8.85, "east": 106.11, "north": 10.37},
+    temporal_extent=["2019-03-01", "2020-12-31"],
+    bands=["B3"]
+)
+
+toc._pg.arguments['featureflags'] = {'byoc_collection_id': byoc_collection_id}
```

Comment:
Is this a temporary solution? Because me may want to raise that this is not really supported in any other client.

Reply:
Actually we were considering to move it to collection properties, but we perhaps lost track of that:
https://github.com/openEOPlatform/architecture-docs/issues/4#issuecomment-1024133069

Reply:
I added a not that it's experimental and might still change. As I would like to have something up for SRR3 @m-mohr can we merge this and I will follow up on the discussion of putting this in the collection properties?

Reply:
Sure...